packages/resource-pooling/resource-pooling.0.3/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "Jan Rochel <jan@besport.com>"
+authors: "Jan Rochel (BeSport)"
+homepage: "https://github.com/ocsigen/resource-pooling"
+bug-reports: "https://github.com/ocsigen/resource-pooling/issues"
+license: "MIT"
+dev-repo: "https://github.com/ocsigen/resource-pooling.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+remove: ["ocamlfind" "remove" "resource-pooling"]
+depends: [
+  ("lwt" {>= "2.4.7" & < "4.0.0"})
```

Comment:
just to let you know: parenthesis here are optional. It is safe to remove them but you can let them as well.

Reply:
Alright, thanks. I'll remove them in the next update after this one.